php/tests/test.sh

```diff
@@ -12,11 +12,17 @@ tests=( array_test.php encode_decode_test.php generated_class_test.php generated
 
 for t in "${tests[@]}"
 do
-  echo "****************************"
-  echo "* $t"
-  echo "****************************"
+  echo "************************************"
+  echo "* $t - Extension"
+  echo "************************************"
   php -dextension=../ext/google/protobuf/modules/protobuf.so `which phpunit` --bootstrap autoload.php $t
   echo ""
+
+  echo "************************************"
```

Comment:
This is duplicate of what we do to test php implementation.

Reply:
Could you point me in the right direction to run the existing tests? I wasn't able to get the tests to run against the PHP implementation (and the test to fail) without adding this.

Reply:
cd php
composer install
./vendor/bin/phpunit